app/views/error500.scala.html

```diff
@@ -0,0 +1,17 @@
+@(ex: Throwable)
+@main("Internal Server Error") {
+    <div class="gs-container">
+        @fragments.page.header("Error 500", Some("There was an error on this page"))
+
+        <div class="row">
+            <p>
+              Sorry, there was an error on this page. We're aware of the problem. If this error persists, please contact
+              support on <a href="mailto:subscriptions.dev@@theguardian.com">subscriptions.dev@@theguardian.com</a>.
```

Comment:
Agreed - we should get similar emails set up to membership and figure out who needs to be on the end of those.